content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,460 @@ linkTitle: "Workload Manager"
+Management System (WMS) architecture is composed of multiple loosely coupled
+components working together in a collaborative manner with the help of a common
+Configuration Services ensuring reliable service discovery functionality.  The
+modular architecture allows to easily incorporate new types of computing
+resources as well as new task scheduling algorithms in response to evolving user
+requirements.  DIRAC services can run on multiple geographically distributed
+servers which increases the overall reliability and excellent scalability
+properties.  REST server providing language neutral interface to DIRAC service.
+Web portal provides simple and intuitive access to most of the DIRAC
+functionalities including management of computing tasks and distributed data.
+It also has a modular architecture designed specifically to allow easy extension
+for the needs of particular applications.
+
+![Fig. 1 - The DIRAC Web portal](./figure_1.png)
+
+Fig. 1 - The DIRAC Web portal
+
+## How to access the EGI Workload Manager service
+
+There are several options to access the service:
+
+1. Members of a scientific community whose resources pool is already configured
+   in the DIRAC4EGI instance > can use the DIRAC4EGI web portal to access the
+   service, or use DIRAC Client.
+1. Individual researchers who want to do some number crunching for a limited
+   period of time, with a reasonable (not too high) number of CPUs > can use
+   the catch-all VO resource pool (vo.access.egi.eu).  Submit a request for this
+   (CST to check identity and justification of use).
+1. Representatives of a community who want to try DIRAC and EGI > Same as #2.
+1. Representative of a community who wants to request DIRAC for the community's
+   own resource pool > Submit a request via the Marketplace, CST to call back
+   and discuss the details (which resource pool, what number of users, AAI,
+   etc.).
+
+## Getting Started
+
+### Submit a service order via the Marketplace
+
+User can request access to the service submitting a service-order to use the EGI
+HTC service directly from the EOSC or the EGI Marketplace:
+
+EOSC Marketplace:
+[https://marketplace.eosc-portal.eu/services/egi-workload-manager](https://marketplace.eosc-portal.eu/services/egi-workload-manager)
+
+EGI Marketplace:
+[https://marketplace.egi.eu/compute/73-workload-manager.html](https://marketplace.egi.eu/compute/73-workload-manager.html)
+
+Service orders are usually handled within 5 working days by the EGI User Support
+Team on shift.
+
+### Before to start
+
+#### Apply for your user credentials
+
+DIRAC uses X.509 certificates to identify and authenticate users.  These
+certificates are delivered to each individual by trusted certification
+authorities.  If you have a personal certificate issued by a EUGridPMA-certified
+authority you can use it for this tutorial.  Otherwise, IHEP certification
+authority can deliver one for you.  Click on the link Online for User Request on
```

Comment:
Is this missing real links?

Reply:
Yes, actually several of them was missing, thanks!
89ab988d183b03b126671f694eeece42f5a68ac3